examples/inference_pipelines.ipynb

```diff
@@ -294,7 +294,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.17"
+   "version": "3.10.12"
```

Comment:
why this needs to be changed ?

Reply:
it is a mistake
it does not need change, but I had not noticed that it got pushed along with  the ! --> %  change that I made to the pip install command

Reply:
it does not need to be changed
it change happens whenever one makes a change to the notebook, since it goes with the metadata
I have not found a way to make the push of the changes without it changing the metadata

Reply:
I've checked and the change in the metadata does not affect the use of the notebook in any meaningful way.
A person with other versions of Python may still run it and use any other kernel specification for it.